components/tools/OmeroPy/src/omero/install/jvmcfg.py

```diff
@@ -398,6 +398,12 @@ def adjust_settings(config, template_xml,
             ("pixeldata", pixeldata), ("repository", repository))
 
     for name, StrategyType in loop:
+        if name not in options:
+            raise Exception(
+                "Cannot find %s option. Make sure templates.xml was "
+                "not copied from a 5.0.2 or older server" % name)
```

Comment:
There are certainly options which are currently *only* configurable in templates.xml. I'm hesitant to say that we should prevent it from being edited. This makes upgrades very difficult/dangerous. The only solutions I've thought of so far are:
 * generate everything under `/etc` (in which case nothing is hand-editable)
 * put `etc/` into git and manage it like `lib/scripts`